USAGE.md

```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
 instead expands the selection by that much movement similar to visual
 mode in vi.
 
--   `h`: move cursor left one char
--   `j`: move cursor down one char
--   `k`: move cursor up one char
--   `l`: move cursor right one char
--   `w`: move cursor to next word
--   `b`: move cursor to previous word start
--   `$`: move cursor to end of line
--   `^`: move cursor to start of line
--   `G`: go to line, or end of file with no prefix
+-   `h`: move left by columns
+-   `j`: move down by lines
+-   `k`: move up by lines
+-   `l`: move right by columns
+
+-   `w`: move to next word
+-   `b`: move back word
+-   `$`: move to last column
+-   `^`: move to first character
```

Comment:
`w` and `b` can have count modifiers too, and I still find "start of line" and "end of line" easier to understand.